app/models.py

```diff
@@ -1551,17 +1558,26 @@ def serialize(self):
     AuditEvent.created_at,
 )
 
-# Index for type + acknowledged audit event query count - for admin
-# app "Service updates" requests: while the page of audit events
-# will rely on the created_at index scan and is relatively quick,
-# count(*) query executed by `.paginate` is very slow without a
-# dedicated index for objects with multiple events
+
+# DEPRECATED - remove in a migration once supplier update admin app has been updated
 db.Index(
     'idx_audit_events_type_acknowledged',
     AuditEvent.type,
     AuditEvent.acknowledged,
 )
 
+# partial index for supplier_update-monitoring admin views. specifically this
```

Comment:
Is there a problem with using a full index for this?

I still feel that we should either make it general right away or make it explicit that it isn't at the moment by returning a 400 when `earliest_for_each_object` is used with a different event type.